### DISCUSSION TITLE
OpenTelemetry no context propagation with fetch call

### DISCUSSION THREAD

**Discussion by cheemingli** [vercel/next.js#54877] [Help]
### Summary

I am using the experimental OpenTelemetry feature for instrumentation.
When fetching API's I using the `fetch` call I noticed the context isn't being propagated. 
The `traceparent` header for example isn't being send to the API so there is no parent-child relations between the traces.

Will this be added in the future? What is the current suggested way of propagating the headers?

**Comment by joostmeijles** [comment]
I was able to get it working using https://www.npmjs.com/package/opentelemetry-instrumentation-fetch-node (see also https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1021#issuecomment-1702752873).

It would however be useful to have `fetch` instrumented by default when enabling OpenTelemetry.
